e2e/app/tokenbridge.go

```diff
@@ -244,7 +244,7 @@ func bridgeToNative(ctx context.Context, def Definition, toBridge []BridgeTest)
 // waitNativeBridges waits for all native bridge test cases to complete.
 // This is required before bridging back to L1, because the native bridge must be informed that L1 tokens are available.
 func waitNativeBridges(ctx context.Context, def Definition, bridges []BridgeTest) error {
-	ctx, cancel := context.WithTimeout(ctx, 30*time.Second)
+	ctx, cancel := context.WithTimeout(ctx, 2*time.Minute)
```

Comment:
Not sure why this bump was needed - debugged for a little but couldn't figure it out and didn't want to spin for too long